src/index.test.ts

```diff
@@ -60,16 +60,18 @@ test('export', (t): void => {
           '@typescript-eslint/no-non-null-assertion': 'error',
           '@typescript-eslint/no-object-literal-type-assertion': 'error',
           '@typescript-eslint/no-this-alias': ['error', { allowDestructuring: true }],
-          '@typescript-eslint/no-triple-slash-reference': 'error',
           '@typescript-eslint/no-unnecessary-type-assertion': 'error',
           '@typescript-eslint/no-unused-vars': ['error', { args: 'none' }],
           '@typescript-eslint/no-use-before-define': ['error', { functions: false, classes: false, variables: false, typedefs: false }],
           '@typescript-eslint/no-useless-constructor': 'error',
           '@typescript-eslint/no-var-requires': 'error',
           '@typescript-eslint/prefer-function-type': 'error',
+          '@typescript-eslint/prefer-readonly': 'error',
           '@typescript-eslint/promise-function-async': 'error',
           '@typescript-eslint/restrict-plus-operands': 'error',
           '@typescript-eslint/require-array-sort-compare': 'error',
+          '@typescript-eslint/strict-boolean-expressions': 'error',
```

Comment:
This should be controversial. If you ask many JavaScript developers, they'd tell you that they like boolean casting. I am willing to give it up for the benefit of one less bug once in a while.